kolibri/core/tasks/test/test_job_running.py

```diff
@@ -301,3 +314,60 @@ def test_can_cancel_a_job_that_updates_progress(self, inmem_queue):
             assert time_spent < 5
         # and hopefully it's canceled by this point
         assert job.state == State.CANCELED
+
```

Comment:
Would be good to add tests for:

`test_successful_job_cannot_restart` `test_queued_job_cannot_restart` too!

Reply:
Added these. The tests checks for the `JobNotRestartable` exception.